playbooks/eth1_ip.yml

```diff
@@ -2,6 +2,18 @@
 - hosts: all
   become: True
   tasks:
+    - name: Install nmstate when ansible distribution is 'CentOS'
+      command:
+        cmd: yum install nmstate -y
+        warn: false
```

Comment:
Why is this needed?

Reply:
install nmstate on target machine

Reply:
Ok, but why the `warn: false`?

Reply:
```TASK [Install nmstate when ansible distribution is 'CentOS'] **************************************************************************************************************************************************
task path: /home/wenliang/ansible-nmstate/playbooks/eth1_ip.yml:5
[WARNING]: Consider using the yum module rather than running 'yum'.  If you need to use command because yum is insufficient you can add 'warn: false' to this command task or set 'command_warnings=False' in
ansible.cfg to get rid of this message.```

Reply:
I use `warn: false`  to get rid of warning message

Reply:
Why is this the correct way to handle the warning?

Reply:
This is probably permissive, the above `cmd: yum install nmstate -y`  that would generate a warning would no longer generate a warning.

Reply:
What about the other option that the warning suggests? Why not use that one?

Reply:
I have not try the yum module, but it seems that using yum module is better. I will update soon !

Reply:
@tyll ,updated. Thanks for the hint!

Reply:
FYI: There is also a generic package module that would work for even more distribtuions: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/package_module.html

Reply:
We will change from `dnf` module to `package` module, because it works on more distributions